src/app/shared/book.service.ts

```diff
@@ -21,7 +21,11 @@ import { catchError } from 'rxjs/operators';
     }
 
     findBookById(id: any) : Promise<Object> {
```

Comment:
id: any --- can we give proper type here?

Reply:
done